crime-applications-adaptor/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/crimeapply/FullMeansAssessmentMapper.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.laa.crime.applications.adaptor.mapper.crimeapply;
+
+import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.common.AssessmentDetail;
+import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.common.FullMeansAssessment;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+class FullMeansAssessmentMapper {
+
+    private final OutgoingsMapper outgoingsMapper = new OutgoingsMapper();
+
+    @NotNull
+    FullMeansAssessment map(
+            uk.gov.justice.laa.crime.applications.adaptor.model.criminalapplicationsdatastore.general.OutgoingsDetails crimeApplyOutgoingsDetails) {
```

Comment:
I don't think we need a fully qualified class name here, we can use an import.

Reply:
done